redaxo/src/core/pages/setup.php

```diff
@@ -334,6 +352,29 @@
     $fragment->setVar('elements', $formElements, false);
     $content .= $fragment->parse('core/form/checkbox.php');
 
+    $content .= '</fieldset><fieldset><legend>' . rex_i18n::msg('setup_security') . '</legend>';
+
+    $formElements = [];
+
+    $n = [];
+    $n['label'] = '<label>'.rex_i18n::msg('https_activate_redirect_for').'</label>';
+    $n['field'] = $httpsRedirectSel->get();
+    $formElements[] = $n;
+
+    $fragment = new rex_fragment();
+    $fragment->setVar('elements', $formElements, false);
+    $content .= $fragment->parse('core/form/form.php');
+
+    $formElements = [];
+    $n = [];
+    $n['label'] = '<label>' . rex_i18n::msg('hsts_activate') . '</label>';
+    $n['field'] = '<input type="checkbox" name="use_hsts" value="1"' . $hsts_checked . ' />';
```

Comment:
Möglicherweise noch einen Hinweis, was HSTS ist und was es bewirkt

Reply:
Fänd ich gut. Irgendeine seite die es für auch nicht sooo technisch versierte erklärt.

Vllt https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
Oder eben wikipedia

Reply:
> Vllt https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

Finde ich besser, da direkt im ersten Satz die Auswirkung erklärt wird.
Kleiner Nachteil wäre, dass der Eintrag leider nicht in Deutsch verfügbar ist.